sdk-config.json

```diff
@@ -6,6 +6,7 @@
     "authConfig_comment2": "Optional Full paths to Infinity server OAuth endpoints (Only needed if using custom OAuth service)",
     "authConfig_comment3": "Optional attributes are: authorize, token, revoke, authService, silentTimeout",
```

Comment:
Suggest adding iframeLoginUI as an optional attribute (and not have it explicitly listed).  Rather just have info for it in README

Reply:
I don't think we have complete documentation for the sdk-config.json file options in either the SDK Guide or the README. But the SDK Guide info is more complete than what's in the README. And, since the explanation for all of the various options will get pretty lengthy, I suggest that we have the README point to a (new) complete page of sdk-config.json options that we'll put in the **docs** folder. I'll create a story for that doc work but I don't think we need to hold up merging this PR to wait for its completion (once the other comments on this PR have been addressed).